test/Microsoft.Health.Fhir.Shared.Tests.Integration/Features/Operations/FhirOperationDataStoreTests.cs

```diff
@@ -17,16 +17,15 @@
 
 namespace Microsoft.Health.Fhir.Tests.Integration.Features.Operations
 {
-    [Collection(FhirOperationTestConstants.FhirOperationTests)]
```

Comment:
Just for my knowledge, why was the collection attribute removed?

Reply:
I actually didn't intend to take this out, thanks for catching it. Added it back in and included a few more tests in 0002444.